.travis.yml

```diff
@@ -3,6 +3,9 @@ go:
 - go1.10.2
 script:
 - make test
+- npm install -g snyk
+- snyk test --org=cedar
```

Comment:
should we be changing this "org" over to "runtime" or maybe even "runtime-infrastructure"? i know that's a bigger change than this PR, but just wondering. cc: @cji

Reply:
We can, certainly and I like that idea especially as we start taking on fixing snyk reports based on team sustainability goals! Right now they line up with the Teams based in components.heroku.tools. Once those get updated we can move this over very easily.

Reply:
I wonder about this too and I can probably be convinced either way, however my leaning is not to change the orgs. IMO it's a lot of work I think for not very much gain, unless I'm missing some use case for Snyk orgs. I kind of like the separation of orgs by which runtime it pertains to. Also we have Snyk dashboard security status reports by runtime, which can be nice or useful. Ultimately my question here to guide my decision would be: who uses the Snyk dashboard and how do they use it?

I think my order of preferences would be: leave it as is > move everything to org=runtime > move to org=runtime-infra. It's also a lot easier for me to think of components as tied to a runtime than it is to think of it tied to a team.

Reply:
I'd like to see more things as `runtime` than `cedar` or `dogwood`. I agree that scoping it to `-infra` probably is a little too specific.

Reply:
> Ultimately my question here to guide my decision would be: who uses the Snyk dashboard and how do they use it?
Right now I expect the answer to be "no one", unfortunately. People *may* look at it if a build is blocked because they're introducing a new dependency with a vulnerability, but I think for the most part no one is going back and looking at existing vulnerabilities. I think the same is mostly true for the GitHub security alerts we get. This is another sustainability task we should look to address.

Reply:
I've opened a runtime issue to discuss how we should handle acting on Snyk vulnerabilities https://github.com/heroku/runtime-issues/issues/357